### PR TITLE
AST-2294 - fix: list bullets block misaligned with page container.

### PR DESCRIPTION
**Pull request by neerajmasai**
### Description

The bullets of list items are not following the content width. Its getting out of area.
### Screenshots

https://d.pr/i/N41dEP

### Types of changes

- Fix: list bullets block misaligned with page container.

### How has this been tested?

- Install Astra.
- Create a page.
- Add a list bullets block.
- Check frontend.

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [x] My code follows accessibility standards 
- [x] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [x] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

**Review by nathanrodrigues2111 — APPROVED** [review]

File: inc/dynamic-css/block-editor-compatibility.php [hunk 1]
```diff
@@ -460,6 +461,18 @@ function astra_load_modern_block_editor_ui( $dynamic_css ) {
 			}
 		}
 	';
+	
+	/* Astra Spectra Gutenberg compatibility css. */
+	if ( $ast_spect_comp ) {
+
+		// List bullets alignment with page container.
+		$dynamic_css .= '
+			.entry-content[ast-blocks-layout] > ul {
```
Comment (imnavanath) [inline inc/dynamic-css/block-editor-compatibility.php:470]:
```suggestion
			.entry-content[ast-blocks-layout] > ul, .entry-content[ast-blocks-layout] > ol {
```

File: inc/dynamic-css/block-editor-compatibility.php [hunk 2]
```diff
@@ -186,6 +186,7 @@ function astra_load_modern_block_editor_ui( $dynamic_css ) {
 	$astra_block_editor_v2_ui = astra_get_option( 'wp-blocks-v2-ui', true );
 	$ast_container_width      = astra_get_option( 'site-content-width', 1200 ) . 'px';
 	$blocks_spacings          = Astra_WP_Editor_CSS::astra_get_block_spacings();
+	$ast_spect_comp           = astra_get_option( 'ast-spect-compatibility', true );
```
Comment (imnavanath) [inline inc/dynamic-css/block-editor-compatibility.php:189]:
```suggestion
	$list_blocks_space           = astra_get_option( 'list-blocks-spacing', true );
```

File: inc/theme-update/astra-update-functions.php [hunk 3]
```diff
@@ -1090,4 +1090,11 @@ function astra_theme_background_updater_3_9_3() {
 		);
 		update_option( 'astra-settings', $theme_options );
 	}
+
+	// Spectra Gutenberg compatibility - backwards compatibility flag.
+	if ( ! isset( $theme_options['ast-spect-compatibility'] ) ) {
+		$theme_options['ast-spect-compatibility'] = false;
```
Comment (imnavanath) [inline inc/theme-update/astra-update-functions.php:1096]:
```suggestion
		$theme_options['list-blocks-spacing'] = false;
```

File: inc/dynamic-css/block-editor-compatibility.php [hunk 4]
```diff
@@ -460,6 +461,18 @@ function astra_load_modern_block_editor_ui( $dynamic_css ) {
 			}
 		}
 	';
+	
+	/* Astra Spectra Gutenberg compatibility css. */
+	if ( $ast_spect_comp ) {
```
Comment (imnavanath) [inline inc/dynamic-css/block-editor-compatibility.php:466]:
```suggestion
	if ( $list_blocks_space ) {
```

File: inc/dynamic-css/block-editor-compatibility.php (hunk 1 above)
Comment (neerajmasai) [reply inc/dynamic-css/block-editor-compatibility.php:470]: Done.

File: inc/dynamic-css/block-editor-compatibility.php (hunk 2 above)
Comment (neerajmasai) [reply inc/dynamic-css/block-editor-compatibility.php:189]: Done.

File: inc/theme-update/astra-update-functions.php (hunk 3 above)
Comment (neerajmasai) [reply inc/theme-update/astra-update-functions.php:1096]: Done.

File: inc/dynamic-css/block-editor-compatibility.php (hunk 4 above)
Comment (neerajmasai) [reply inc/dynamic-css/block-editor-compatibility.php:466]: Done.

File: inc/dynamic-css/block-editor-compatibility.php [hunk 5]
```diff
@@ -186,6 +186,7 @@ function astra_load_modern_block_editor_ui( $dynamic_css ) {
 	$astra_block_editor_v2_ui = astra_get_option( 'wp-blocks-v2-ui', true );
 	$ast_container_width      = astra_get_option( 'site-content-width', 1200 ) . 'px';
 	$blocks_spacings          = Astra_WP_Editor_CSS::astra_get_block_spacings();
+	$list_blocks_space           = astra_get_option( 'list-blocks-spacing', true );
```
Comment (imnavanath) [inline inc/dynamic-css/block-editor-compatibility.php:189]:
```suggestion
	$blocks_spacings          = Astra_WP_Editor_CSS::astra_get_block_spacings();
	$list_blocks_space         = astra_get_option( 'list-blocks-spacing', true );
```

**Review by imnavanath — APPROVED** [review]

**Pull request merged**